styles/index.css

```diff
@@ -44,9 +44,8 @@
 }
 
 .arguments dd {
-  display: block;
+  display: inline-block;
   margin-left: 40px;
-  float: left;
   width: 80%;
   padding: 0;
   margin: 0;
```

Comment:
Are you sure we need these changes?
It might be because of the comment above, the "arguments" class should only be applied to the Argument section for new packages parsed